.golangci.yml

```diff
@@ -134,3 +134,16 @@ issues:
         - gochecknoglobals
         - err113
         - wrapcheck
+
+    # Exclude some linters from running on tests files.
+    - path: e2e/.*|e2e/framework/.*
```

Comment:
Hmm - do we not want to enforce the same coding standards across our e2es? I can understand excluding gosec.. not as sure about the others. 
cc @damdo